src/assets/stylesheets/colors.scss

```diff
@@ -5,3 +5,8 @@
 
   Please refrain from adding code that generates CSS.
 */
+
+$sky-blue: #56baec;
+$grey: #f6f6f6;
+$sky-blue-disabled: #94a8b280;
+$sky-blue-on-hover: #79c7ee;
```

Comment:
It's good practice to end all files with a newline. There is a technical reason related to some standards of text files, but also it helps get a cleaner `git diff` when adding new code on the file